admin/js/menu-page.js

```diff
@@ -1,159 +1,25 @@
-                $('.opinionstage-dashboard-left').show()
-              }
-              self.cache.$messageNoItemsFound.show()
-            }
-          } else {
-            self.cache.$failedLoadItemsMessage.show()
-          }
-        }
-      })
-    },
-
-    bindEvents: function () {
-      var self = this
-      self.cache.$buttonLoadMore.on('click', function (e) {
-        e.preventDefault()
-        self.loadItems(false)
-      })
-
-      $('#itemList').on('change', function (e) {
-        e.preventDefault()
-        self.cache.$loadtPage = 1
-        self.cache.$searchField.val('')
-        self.cache.searchTerm = ''
-        self.cache.$widgetType = $(this).val()
-        self.loadItems()
-      })
-
-      self.cache.$searchField.on("keyup", function (e) {
-        if (e.key === 'Enter') {
-          self.cache.searchTerm = $(this).val()
-          self.cache.$loadtPage = 1
-          self.loadItems()
-        }
-      })
-
-      $('body').on('click', '.opinionstage-open-modal', function (e) {
-        e.preventDefault()
-        
-        var $modalPublishedDetails = $('#published-item-details')
-        var $modalEditLink = $('#opinionstage-modal-edit-link')
-        var $modalDraftDetails = $('#draft-item-details')
-        var $this = $(this)
-        if ($this.data('is-draft')) {
-          $modalPublishedDetails.hide()
-          $modalEditLink.attr('href', $this.data('edit-url'))
-          $modalDraftDetails.show()
-        } else {
-          $modalDraftDetails.hide()
-          $modalEditLink.attr('href', '#')
-          $modalPublishedDetails.show()
-          $(self.cache.$modalShortcodeTextarea).val($this.data('shortcode'))
-        }
-
-        
-        self.cache.$modal.fadeIn()
-      })
-
-      self.cache.$modal.on('click', function (e) {
-        if ($(e.target).is(self.cache.$modal)) {
-          self.cache.$modal.fadeOut(function () {
-            $(self.cache.$modalShortcodeTextarea).val('')
-          })
-        }
-      })
-      self.cache.$closeModalButton.on('click', function (e) {
-        self.cache.$modal.fadeOut(function () {
-          $(self.cache.$modalShortcodeTextarea).val('')
-        })
-      })
+  if( $('[data-opinionstage-content-popup-template]').length > 0 ) {
+      var $modal = $('#opinionistage-my-items-page-modal-wrapper')
+    
+    function showModal( selectedWidgetData){
+      $modal.find('#opinionstage-widget-shortcode').val(selectedWidgetData.shortcode)
+      $modal.fadeIn()
+    }
 
-      $("body").on("click", '[data-copy-text-from]', function (e) {
-        e.preventDefault()
-        var t = $(this).data().copyTextFrom
-        $("[" + t + "]")[0].select()
-        document.execCommand("copy")
+    $('#opinionstage-dialog-close').on('click', function (e) {
+      e.preventDefault()
+      $modal.fadeOut(function () {
+          $(this).find('#opinionstage-widget-shortcode').val('')
       })
-    },
+    })
+  OpinionStage.contentPopup.renderMyItemsPageView(showModal)
```

Comment:
on my items page, it's not a content popup anymore, further usage of this naming will lead to confusion, suggest we use another naming and implement my items page and content popup as separate entry points: `OpinionStage.widgetList.show(...)` for my items page and `OpinionStage.contentPopup.open()`